.github/workflows/auto-docs.yml

```
@@ -0,0 +1,373 @@
+        description: 'Force documentation update'
+        required: false
+        default: 'false'
+        type: boolean
+
+jobs:
+  auto-documentation:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)